### DISCUSSION TITLE
Is there an alternative documentation home?

### DISCUSSION THREAD

**Discussion by johnwheeler** [fabricjs/fabric.js#8923] [General]
The JSDocs do not have a consolidated list of members and some classes have hundreds of properties. It'd be nice to see them without having to scroll through the whole page

**Comment by ShaMan123** [comment]
Which version are you working on?
v6 is typed

**Comment by asturur** [comment]
@johnwheeler we are working on a website with proper jsdocs. it requires a lot of time and it is always pushed back for new code.
We will get there at some point
